src/app/shared/helper-services/export-to-template.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import * as XLSX from 'xlsx';
+import { UtilityMeterdbService } from 'src/app/indexedDB/utilityMeter-db.service';
+import { IdbUtilityMeter } from 'src/app/models/idb';
+@Injectable({
```

Comment:
Don't provide in root scope, provide directly in using module

Reply:
We are using root scope in this project. Haven't broken up code into modules